net/samba4/Portfile

```diff
@@ -1,36 +1,53 @@
 PortSystem 1.0
 
 name			samba4
-version			4.0.0tp5
-revision		5
+version			4.13.0
 categories		net
 platforms		darwin
 maintainers		nomaintainer
 license			GPL-3
-description		SMB/CIFS server and client, ALPHA release
+description		SMB/CIFS server and client
 long_description	Samba is an software suite that provides seamless file \
-			and print services to SMB/CIFS clients. \
-			This is an ALPHA release!
+			and print services to SMB/CIFS clients.
 
 homepage		http://www.samba.org/
-master_sites		http://download.samba.org/samba/ftp/samba4/ \
-			ftp://de.samba.org/samba.org/samba4/
+master_sites		https://download.samba.org/pub/samba/stable
 distname		samba-${version}
-checksums           md5     dbc2e04455d67b01d2020808065b96df \
-                    sha1    8897cc7a824a594b3543245cdbbceaea32e9e08d \
-                    rmd160  1870f67aa851751e381a3b03dadfa1d0afa28a12
+checksums       rmd160  0e0efa2be30af3df3cce92b00496b296cc4c2a0d \
+                sha256  f11d52aee8db0aa50ac614143f2ccefead29e8f8ff585358792221557cade7f6
 
-depends_lib		port:gnutls port:readline port:libiconv port:popt port:sqlite3 port:zlib port:gettext
+depends_build   port:python37
+depends_lib		port:gnutls port:readline port:libiconv port:popt port:zlib port:gettext port:libarchive port:openldap port:gpgme
 
-worksrcdir		samba-${version}/source
+worksrcdir		samba-${version}
 
-configure.args		--with-fhs \
-			--mandir=${prefix}/share/man \
-			--with-libiconv=${prefix} --with-readline=${prefix} --with-sqlite3
+platform darwin {
+    depends_build-append \
+                port:cctools
+    depends_lib-append \
+                port:popt \
+                port:jansson
+}
+
+patch.pre_args  -Np1
+platform darwin {
+    patchfiles-append patch-fix-st_atim.diff patch-no-xsltproc.diff
+    configure.ldflags-append -v
+}
+
+configure.cmd   ${prefix}/bin/python3.7 ./buildtools/bin/waf configure
+build.cmd       ${prefix}/bin/python3.7 ./buildtools/bin/waf -v
```

Comment:
Make a variable to specify the python variable once and use it all build and destroot commands. Also it should be using 3.8.